rotkehlchen/api/v1/schemas.py

```diff
@@ -1880,10 +1880,15 @@ class IgnoredAssetsHandling(SerializableEnumMixin):
     SHOW_ONLY = auto()
 
 
-class AssetsPostSchema(DBPaginationSchema, DBOrderBySchema):
+class AssetsPostSchema(DBPaginationSchema, DBOrderBySchema, OptionalEthereumAddressSchema):
```

Comment:
if you defined the fields then this `OptionalEthereumAddressSchema` is not needed. Indeed the type fields are different